drivers/platform/maxim/max32650/maxim_usb_uart.c

```diff
@@ -0,0 +1 @@
+../common/maxim_usb_uart.c
```

Comment:
ok I see @CiprianRegus opened #2137 meaning I will have to delete symlinks around here as well